frame/benchmarking/src/lib.rs

```diff
@@ -50,1883 +50,226 @@ pub use sp_std::{self, boxed::Box, prelude::Vec, str, vec};
 pub use sp_storage::TrackedStorageKey;
 pub use utils::*;
 
-/// Whitelist the given account.
-#[macro_export]
-macro_rules! whitelist {
-	($acc:ident) => {
-		frame_benchmarking::benchmarking::add_to_whitelist(
-			frame_system::Account::<T>::hashed_key_for(&$acc).into(),
-		);
-	};
-}
+pub mod v1;
```

Comment:
So when I use `frame_benchmarking::v1` instead of `frame_benchmarking` in the assets benchmark, it does not work.  
The cargo doc shows `frame_benchmarking::v1` pretty empty as well.

Reply:
resolved with latest commit, though dealing with a weird CI issue now